example/rater/generated_answer.ipynb

```diff
@@ -143,7 +144,7 @@
+      "                                        guided_prompt_template=GuidedPrompt(instruction=\"\\n            Compare two answers: a generated answer and a grounding answer based on a provided context and question.\\n            There are few annotated examples below, consisting of context, question, grounding answer, generated answer, explanation and label.\\n            If generated answer is better, you should give a higher score and vise versa. Check label to score dictionary: [('accept', 1.0), ('equivalent', 0.0), ('reject', -1.0)].\\n            Your response should only focus on the unlabeled sample, including two fields: explanation and label (one of ['accept', 'equivalent', 'reject']).\\n            \", examples=[Context(context='Early computers were built to perform a series of single tasks, like a calculator.', question='Did early computers function like modern calculators?', grounding_answer='No. Early computers were used primarily for complex calculating.', generated_answer='Yes. Early computers were built to perform a series of single tasks, similar to a calculator.', explanation='The generated answer is better because it correctly figures out early computers was used to perform single tasks akin to calculators and asigned a score of 1.0.', label='accept')]),\n",
```

Comment:
If there a way to better print this chunk to be more claer? Not have to use pprint.

Reply:
I think it is because a `__repr__` is already defined in `pydantic.BaseModel` (baseclass of `PromptTemplate`).